src/app/pages/card-import/card-import.page.html

```diff
@@ -1,8 +1,15 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="primary">
+      <ion-back-button></ion-back-button>
+    </ion-buttons>
+  </ion-toolbar>
+</ion-header>
 
 <ion-content class="card-image">
     <ion-card button (click)="openCameraFront()" style="height:auto;weight:auto;">
-      <img ng-show="isFrontCaptured" [src]="card.frontImg" alt=""/>
```

Comment:
Aha, these are those changes you made. Definitely want to check them out in action.

Reply:
Yes. This changes three concepts:
1. Instead of saving images directly into the card object on every capture, save it to a local variable then save it to the card when the save button is clicked.
2. Instead of an extra boolean that tells us if the front image had been captured, just check for said local variable to determine whether the captured image should be displayed.
3. Instead of `*ngShow`, which will only visually remove the element while still incurring processing costs, use `*ngIf` which will actually prevent the element from being generated.